components/svg/ipx.tsx

```diff
@@ -13,7 +13,7 @@ const IPX: FC<SVGProps> = ({ maxHeight, maxWidth, ...props }) => (
       fillRule="evenodd"
       clipRule="evenodd"
       d="M10.3804 0.666504C12.1955 0.666504 13.6667 2.14345 13.6667 3.96509V7.35803H10.2864C8.4715 7.35803 7.00004 5.88108 7.00004 4.05944V0.666504H10.3804ZM16.953 7.35803C15.1382 7.35803 13.6667 8.83498 13.6667 10.6566V14.0496H17.047C18.8619 14.0496 20.3334 12.5726 20.3334 10.751V7.35803H23.7137C25.5286 7.35803 27 5.88108 27 4.05944V0.666504H23.6197C21.8048 0.666504 20.3334 2.14345 20.3334 3.96509V7.35803H16.953ZM17.047 20.7411C18.8619 20.7411 20.3334 22.218 20.3334 24.0397V27.4326H16.953C15.1382 27.4326 13.6667 25.9557 13.6667 24.134V20.7411H17.047ZM7.00004 14.0496H3.6197C1.80484 14.0496 0.333374 12.5726 0.333374 10.751V7.35803H3.71371C5.52858 7.35803 7.00004 8.83498 7.00004 10.6566V14.0496ZM7.00004 14.0496V17.4425C7.00004 19.2641 8.4715 20.7411 10.2864 20.7411H13.6667V17.3481C13.6667 15.5265 12.1955 14.0496 10.3804 14.0496H7.00004ZM23.6197 14.0496C21.8048 14.0496 20.3334 15.5265 20.3334 17.3481V20.7411H23.7137C25.5286 20.7411 27 19.2641 27 17.4425V14.0496H30.3804C32.1952 14.0496 33.6667 12.5726 33.6667 10.751V7.35803H30.2864C28.4715 7.35803 27 8.83498 27 10.6566V14.0496H23.6197Z"
-      fill="#0053DB"
+      fill="#FFF"
```

Comment:
when u need change a color in a SVG Element, put `currentColor` on the `fill` props